README.md

```diff
@@ -26,15 +26,14 @@ Currently, the minimum supported Rust version is `1.48.0`.
 
 ## Documentation
 
-### Development Branch
-
-- [Book](https://gtk-rs.org/gtk4-rs/git/book)
-- [GTK](https://gtk-rs.org/gtk4-rs/git/docs/gtk4/)
-- [GDK](https://gtk-rs.org/gtk4-rs/git/docs/gdk4/)
-- [GSK](https://gtk-rs.org/gtk4-rs/git/docs/gsk4/)
-- [GDK Wayland](https://gtk-rs.org/gtk4-rs/git/docs/gdk4_wayland/)
-- [GDK X11](https://gtk-rs.org/gtk4-rs/git/docs/gdk4_x11/)
-- [GTK Macros](https://gtk-rs.org/gtk4-rs/git/docs/gtk4_macros/)
+- [Examples](https://github.com/gtk-rs/gtk4-rs/tree/master/examples)
+- Book [Stable]/[Development](https://gtk-rs.org/gtk4-rs/book/git/index.html)
+- GTK [Stable]/[Development](https://gtk-rs.org/gtk4-rs/docs/git/gtk4/)
+- GDK [Stable]/[Development](https://gtk-rs.org/gtk4-rs/docs/git/gdk4/)
+- GSK [Stable]/[Development](https://gtk-rs.org/gtk4-rs/docs/git/gsk4/)
+- GDK Wayland [Stable]/[Development](https://gtk-rs.org/gtk4-rs/docs/git/gdk4_wayland/)
+- GDK X11 [Stable]/[Development](https://gtk-rs.org/gtk4-rs/docs/git/gdk4_x11/)
+- GTK Macros [Stable]/[Development](https://gtk-rs.org/gtk4-rs/docs/git/gtk4_macros/)
```

Comment:
I think we should just link to the main docs instead of explicitly giving a link to each crate here. So something like 
```
- gtk4-rs [Stable]/[Development](https://gtk-rs.org/gtk4-rs/docs/git/) should be enough I guess

Reply:
That link does not work

Reply:
Of course it doesn't because it's supposed to be https://gtk-rs.org/gtk4-rs/git/docs/

Reply:
Fixed